on-premises/requirements.md

```diff
@@ -1,19 +1,69 @@
-# Requirements for Rollbar On-Premises
+# Configuration Requirements for Rollbar On-Premises
 
 ## Hardware
 
-### Recommended for single-host install
-
-The single-host install should work well with the following specs - The AWS EC2
-equivalent of m4.xlarge.
+The single-host install should work well with the following specs.  This
+is the equivalent an AWS EC2 m4.xlarge instance.
 
 - 4+ core
 - 16GB RAM
 - 100+GB disk
 
 ## Operating System and Packages
 
-- Ubuntu 14.04
+Currently we've only verified our on-premises product to work on the
+following, though we're working on adding more Linux operating system
+distributions and versions.  Due to the fact that we're running on
+Docker, this will never run natively on Windows or Mac hosts.  You'll
+have to setup some sort of virtual machine(s) in those instances.
+
+- Ubuntu 14.04 LTS
 - Docker v1.9.1+
   - Tested with Docker 1.9.1, 1.10.1
 - Docker Compose 1.6.1
+
+## Email
+
+In some of the environments we've encountered, the built-in SMTP server
+has problems sending mail.  Generally this is from things like bad IP
+address reputation or lack of good SPF and DKIM setups.  Needless to
+say, you'll almost certainly want to run your own SMTP server, or hook
+in to a service like Mailgun or Amazon SES.
+
+## Network Configuration
+
+We assume that you will at least have a publicly facing network and a
+private network for the docker host or hosts (if not more).  The
+following ports need to be open, both on the public and private
+networks:
+
+* outbound ports: 25 (smtp), 80 (http) && 443 (https)
```

Comment:
Does port 25 need to be open on the public network even if you are using an smtp server that is on your private network?

Also, do we only support port 25 for smtp or do we also work with smtp over TLS? (typically port 465)